src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -173,15 +175,15 @@ public InmateDetail createPrisoner(@RequestBody @NotNull @Valid final RequestToC
         @ApiResponse(responseCode = "403", description = "Forbidden - user not authorised to release a prisoner.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "*** BETA *** Releases a prisoner from their current prison location. Must be an active prisoner in currently inside a prison, requires the RELEASE_PRISONER role")
+    @Operation(summary = "Releases a prisoner from their current prison location. Must be an active prisoner in currently inside a prison, requires the RELEASE_PRISONER role")
```

Comment:
Actually I think this endpoint is BETA since it has never been formally tested - all we know is the restricted patients version is used and presumably works